ext/storage_sources/s3_store/CMakeLists.txt

```diff
@@ -7,34 +7,34 @@ set_property(TARGET aws_bucket_util PROPERTY POSITION_INDEPENDENT_CODE ON)
 target_include_directories(
     aws_bucket_util
     PUBLIC
-        ${CMAKE_CURRENT_SOURCE_DIR}   
+        ${CMAKE_CURRENT_SOURCE_DIR}
 )
 
 target_link_libraries(
     aws_bucket_util
     PUBLIC
         aws-sdk::s3-crt
-        aws-sdk::core   
+        aws-sdk::core
 )
 
 # Build the S3 extension as a library.
 set(sources "s3_store.cxx")
```

Comment:
Is there a need to set this variable? I see it only being used once.

Reply:
Will be done in the next ticket. The source file has been renamed to `s3_storage_source.cpp` in develop